### ISSUE TITLE
About class CrossValidation issues

### ISSUE THREAD

**Issue by XieXiao1997** [Project-MONAI/MONAI#3204]
First,Thanks to the Monet Project for providing clinicians with a concise use of deep learning!
When I try to use the Cross-validation method to load data, the training set and validation set will use different transforms to process data, so I would like to ask how to deal with this situation.
Thank you for your reply.
